Makefile

```diff
@@ -47,18 +47,24 @@ endif
 export GOFLAGS:=$(GOFLAGS)
 export CGO_ENABLED=0
 
+BINDIR:=/usr/bin/
+
+build: out
+	@echo "> Building..."
+	$(GOCMD) build -ldflags "-s -w -X 'github.com/buildpacks/pack.Version=${PACK_VERSION}' -extldflags ${LDFLAGS}" -trimpath -o ./out/$(PACK_BIN) -a ./cmd/pack
+
 all: clean verify test build
 
+install:
+	mkdir -p ${DESTDIR}${BINDIR}
+	cp ./out/$(PACK_BIN) ${DESTDIR}${BINDIR}/
+
```

Comment:
Is this necessary? If so, can we add a comment as to why and where we can "learn more".

Reply:
Not `needed` but I figure that users want `pack` on the path after running and `apt-get install`.

Reply:
Ok, so it is _somewhat_ necessary for `apt-get` to work the way we want it to. 

How does it know to run this target? Is there a convention or can we name it something else? Are there some docs or a place we use it in our workflow? Those are the things I think should be answer/noted in the target.